pro/src/new_components/CreditInput/CreditInput.tsx

```diff
@@ -5,12 +5,14 @@ import TextInput from 'components/layout/inputs/TextInput/TextInput'
 export interface CreditInputProps {
   credit: string
   updateCredit: (credit: string) => void
+  handleNext?: (event: React.KeyboardEvent<HTMLInputElement>) => void
   extraClassName?: string
 }
 
 export const CreditInput: FunctionComponent<CreditInputProps> = ({
   credit,
   updateCredit,
+  handleNext,
```

Comment:
Ce composant est un wrapper de TextInput, les propriété transferer de l'un à l'autre devrait etre similaire.
```suggestion
  onKeyDown,
```